projects/dex-ui/src/components/Frame/Grid/useWiggle.tsx

```diff
@@ -0,0 +1,327 @@
+import { RefObject, useEffect, useRef } from "react";
+import { roundPathCorners } from "./roundCorners";
+import throttle from "lodash/throttle";
+import { Queue } from "./Queue";
+import useRequestAnimationFrame from "./useAnimationFrame";
+import {
+  debugPoint,
+  debugLine,
+  getCorners,
+  arePointsColinear,
+  flattenDiagonals,
+  getClosestPoint,
+  isPointInBox,
+  isPointOnBox,
+  pointsEq,
+  snapToGrid,
+  getTraceables,
+  converToSvgPoint,
+  dist,
+  getIntersection
+} from "./utils";
+
+const DEBUG = false;
```

Comment:
Can we use the logger here? And elsewhere in the file where we output debug information.